packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -24,6 +24,16 @@ export const numberOverrideProcessor = (
   return parseFloat(value);
 };
 
+export const displayNameOverrideProcessor = (
+  value: any,
+  context: FieldOverrideContext,
+  settings?: StringFieldConfigSettings
+) => {
+  // clear the cached display name
+  delete context.field?.state?.displayName;
```

Comment:
Ahhh -- the magic!

Reply:
yea, well this was done outside in the field overrides code, but after all everything was processed. Now only done if the display name is changed